src/main/java/model/QualityIndicator.java

```diff
@@ -0,0 +1,117 @@
+package main.java.model;
+
+import java.util.ArrayList;
+
+/**
+ * A class that represents the indicators quality of the tools in comparison with iPlasma and PDM
+ */
+
+public class QualityIndicator {
+
+	private int PMDDCI, PMDDII, PMDADCI, PMDADII, iPlasmaDCI, iPlasmaDII, iPlasmaADCI, iPlasmaADII;
+	private ArrayList<ExcelRow> excelRows;
+	
+	/**
+	 * Creates a IndicatorQuality object based on:
+	 *
+	 * @param PMDDCI - DCI Indicator - number of faults correctly identified by the PMD tool
+	 * @param PMDDII - DII Indicator - number of faults incorrectly identified by the PMD tool
+	 * @param PMDADCI ADCI Indicator - number of absence of faults correctly identified by the PMD tool
+	 * @param PMDADII - ADII Indicator - number of absence of faults incorrectly identified by the PMD tool
+	 * @param iPlasmaDCI - DCI Indicator - number of faults correctly identified by the iPlasma tool
+	 * @param iPlasmaDII - DII Indicator - number of faults incorrectly identified by the iPlasma tool
+	 * @param iPlasmaADCI ADCI Indicator - number of absence of faults correctly identified by the iPlasma tool
+	 * @param iPlasmaADII - ADII Indicator - number of absence of faults incorrectly identified by the iPlasma tool
+	 * 
+	 * A IndicatorQuality object also calculate the Quality Indicators from the Excel file
+	 */
+	public QualityIndicator(ArrayList<ExcelRow> excelRows) {
```

Comment:
Na verdade, acabei de reparar que não adicionasse o javadocs o único parâmetro que realmente está no construtor haha Não esquece de adicionar o javadocs pro `excelRows`.

Reply:
done

Reply:
not pushed here...